src/applications/personalization/profile360/containers/PaymentInformation.jsx

```diff
@@ -172,44 +182,62 @@ class PaymentInformation extends React.Component {
     } else if (paymentInformation.error) {
       content = <LoadFail information="payment" />;
-    } else if (!directDepositIsSetUp) {
-      return null;
     } else {
       content = (
         <>
           <div className="vet360-profile-field">
             <ProfileFieldHeading
-              onEditClick={() => this.handleLinkClick('edit', 'bank-name')}
+              onEditClick={
+                directDepositIsSetUp &&
+                (() => this.handleLinkClick('edit', 'bank-name'))
+              }
             >
               Bank name
             </ProfileFieldHeading>
-            {paymentAccount.financialInstitutionName}
+            {directDepositIsSetUp
+              ? paymentAccount.financialInstitutionName
+              : this.renderSetupButton('bank name', 'bank-name')}
           </div>
           <div className="vet360-profile-field">
             <ProfileFieldHeading
-              onEditClick={() => this.handleLinkClick('edit', 'account-number')}
+              onEditClick={
+                directDepositIsSetUp &&
+                (() => this.handleLinkClick('edit', 'account-number'))
+              }
             >
               Account number
             </ProfileFieldHeading>
-            {paymentAccount.accountNumber}
+            {directDepositIsSetUp
+              ? paymentAccount.accountNumber
+              : this.renderSetupButton('account number', 'account-number')}
           </div>
           <div className="vet360-profile-field">
             <ProfileFieldHeading
-              onEditClick={() => this.handleLinkClick('edit', 'account-type')}
+              onEditClick={
+                directDepositIsSetUp &&
+                (() => this.handleLinkClick('edit', 'account-type'))
+              }
             >
               Account type
             </ProfileFieldHeading>
-            {paymentAccount.accountType}
+            {directDepositIsSetUp
+              ? paymentAccount.accountType
+              : this.renderSetupButton(
+                  'account type (checking or savings)',
+                  'account-type',
+                )}
           </div>
-          <p>
-            <strong>Note:</strong> If you think you’ve been the victim of bank
-            fraud, please call us at{' '}
-            <a href="tel:1-800-827-1000" className="no-wrap">
-              800-827-1000
-            </a>{' '}
-            (TTY: <span className="no-wrap">800-829-4833</span>
-            ). We’re here Monday through Friday, 8:00 a.m. to 9:00 p.m.
-          </p>
+          {directDepositIsSetUp && (
+            <p>
+              <strong>Note:</strong> If you think you’ve been the victim of bank
+              fraud, please call us at{' '}
+              <a href="tel:1-800-827-1000" className="no-wrap">
+                800-827-1000
+              </a>{' '}
+              (TTY: <span className="no-wrap">800-829-4833</span>
+              ). We’re here Monday through Friday, 8:00 a.m. to 9:00 p.m.
```

Comment:
Does the timezone need to be specified here? I noticed Eastern Time is used elsewhere in `vets-website`.

Reply:
Oh good question. I'll ask my team and add that if I hear back. But since it's been like this for weeks I'll leave it for now.